packages/ncids-js/src/components/usa-combo-box/__tests__/combo-box.events.test.ts

```diff
@@ -484,5 +496,13 @@ describe('Combo box - Events', () => {
 			comboBox: comboBoxEl,
 			inputValue: 'aa',
 		});
+
+		await user.keyboard('[Escape]');
+		expect(textClearedEvent).toHaveBeenCalledTimes(1);
+		expect(textClearedEvent.mock.calls[0][0].detail).toEqual({
+			comboBox: comboBoxEl,
+			previousInputValue: 'aa',
+			selected: selected,
+		});
```

Comment:
Do you need a test for when there is no selection and you clear the text?

Reply:
After discussion, we're going to deal with this in another ticket.